website/docs/components/form/super-select/partials/code/how-to-use.md

```diff
@@ -0,0 +1,231 @@
+
+```handlebars
+<Hds::Form::SuperSelect::Single::Field
+  @onChange={{this.noop}}
+  @selected={{this.SELECTED_CLUSTER_SIZE_OPTION}}
+  @options={{this.CLUSTER_SIZE_OPTIONS}}
+  as |F|
+>
+  <F.Label>Size</F.Label>
+
+  <F.Options>
+    {{#let F.options as |option|}}
+      <div class="doc-super-select-option-rich-content">
+        <p class="doc-super-select-option-rich-header">
+          <strong>{{option.size}}</strong>
+          <strong>{{option.price}}</strong>
+        </p>
+        <p>{{option.description}}</p>
+      </div>
+    {{/let}}
+  </F.Options>
+</Hds::Form::SuperSelect::Single::Field>
+```
+
+#### SuperSelect::Multiple
+
+Note: By default all the option content will display in the selected item “tags” which display in the trigger. To simplify the content in these tags, use `@selectedItemComponent` to specify a custom component.
+
+**Custom “tag” component example:**
+
+```markup
+<span>
+  {{@option.size}}
+</span>
+```
+
+**Component invocation:**
+
+```handlebars
+<Hds::Form::SuperSelect::Multiple::Field
+  @onChange={{this.noop}}
+  @selected={{this.SELECTED_CLUSTER_SIZE_OPTIONS}}
+  @selectedItemComponent={{component "power-select/selected-option"}}
+  @options={{this.CLUSTER_SIZE_OPTIONS}}
+  as |F|
+>
+  <F.Label>Size</F.Label>
+  <F.Options>
+    {{#let F.options as |option|}}
+      <div class="doc-super-select-option-rich-content">
+        <p class="doc-super-select-option-rich-header">
+          <strong>{{option.size}}</strong>
+          <strong>{{option.price}}</strong>
+        </p>
+        <p>{{option.description}}</p>
+      </div>
+    {{/let}}
+  </F.Options>
+</Hds::Form::SuperSelect::Multiple::Field>
+```
+
+#### Helper text
+
+You can add extra information to the field using helper text. When helper text is added, the component automatically adds an `aria-describedby` attribute to the Super Select control, associating it with the automatically generated `ID` of the helper text element.
+
+```handlebars
+<Hds::Form::SuperSelect::Single::Field
+  @onChange={{this.noop}}
+  @options={{this.GROUPED_OPTIONS}}
+  @selected={{this.SELECTED_GROUPED_OPTION}}
+  @ariaLabel="Label"
+  as |F|
+>
+  <F.Label>Target infrastructure</F.Label>
+  <F.HelperText>The target infrastructure is where you want to deploy your apps.</F.HelperText>
+  {{F.options}}
+</Hds::Form::SuperSelect::Single::Field>
+```
+
+#### Extra content in label and helper text
+
+!!! Warning
+
+If a link is used within a label, helper text, or error text, it will not be presented as a link to the user with a screen reader; only the text content is read out. As such, care should be used when considering this feature. If needing to use a link, include a screen reader-only message that informs the user that some help text includes links, and additional keyboard exploration may be required.
+!!!
+
+The `Label` and `HelperText` contextual components used in the Field component yield their content. This means you can also pass structured content.
```

Comment:
General question; these are stylized as `Label` and `HelperText` while other instances are plaintext label, and helper text. Should they be the same?

Reply:
Probably yes. Let me compare and see what we've done elsewhere...

Reply:
Looking at it more closely I'm thinking that "Label" and "HelperText" here explicitly refer to the HDS components while in the other areas they could be considered as more general terms. Other form component documentation seems to follow this pattern so I will leave as is.